Texstyle/Texstyle/Text.swift

```diff
@@ -47,48 +58,22 @@ public final class Text {
         }
     }
 
-    /// Initialize the text with passed string and styles for appropriate states.
-    ///
-    /// - Parameters:
-    ///   - value: The string for style and substyles.
-    ///   - styles: The dictionary with states and text styles.
-    public init(value: String, styles: [ControlState: TextStyle]) {
-        self.value = value
-        self.styles = styles
-    }
-
-    /// Initialize the text with passed string and styles for appropriate states. Returns nil if there is no value.
-    ///
-    /// - Parameters:
-    ///   - value: The string for style and substyles.
-    ///   - styles: The dictionary with states and text styles.
-    public convenience init?(value: String?, styles: [ControlState: TextStyle]) {
-        if let value = value {
-            self.init(value: value, styles: styles)
-        }
-        else {
-            return nil
-        }
-    }
-
     /// Adds the substyle for passed range.
     ///
     /// - Parameters:
     ///   - substyle: The substyle for passed range.
     ///   - range: The range for substyle applying.
-    ///   - state: The state for passed substyle. Default is normal.
-    public func add(_ substyle: TextStyle, at range: Range<String.Index>, for state: ControlState = .normal) {
-        add(substyle, at: .init(range, in: value), for: state)
+    public func add(_ substyle: TextStyle, at range: Range<String.Index>) {
```

Comment:
I saw your @available flag, but what's about these `add` functions? I guess we can break API for all cases because states were used rarely and this framework is not so popular.